docs/News.md

```diff
@@ -1,3 +1,8 @@
+## Dec 10, 2019
+* **New Feature**: Show gene panel information on the _Patient View_ page. This helps to distinguish between whether a mutation in a particular gene was not present in the sample or not profiled ([example](https://www.cbioportal.org/patient?caseId=P-0001613&studyId=lung_msk_2017)):
```

Comment:
Wondering if we can find an example with a mutation in one example but not sequenced in another?

Reply:
that is in this one right or did I misunderstand? By default it's filtered:

![Screen Shot 2019-12-10 at 7 14 44 PM](https://user-images.githubusercontent.com/1334004/70580320-5b04e500-1b81-11ea-913c-ef47ceaaafa3.png)

Reply:
it might make sense to show mutations profiled in any sample by default?

Reply:
I definitely agree we should show all mutations by default - I actually made an issue about this the other day :) (#6890 )  If possible, would be great to change this before we make the announcement.

Also how about this case as an example: https://www.cbioportal.org/patient?caseId=P-0002208&studyId=msk_impact_2017
The gene that isn't on one of the panels has a likely oncogenic mutation which I think makes for a nice example.

Reply:
For what it's worth, I also found another issue (#6891) - not sure if it's practical to fix that before this news announcement, but if possible that would be great.

Reply:
Thanks Tali! I like that example as well. I think we can wait with the announcement until we make those fixes. @jjgao do u agree to change the default behavior?

Reply:
I agree with @tali.